v0/destinations/marketo/transform.js

```diff
@@ -297,4 +297,15 @@ const process = async event => {
   return response;
 };
 
-exports.process = process;
+const processRouterDest = input => {
+  const inputs = input;
+  inputs.batched = false;
+  inputs.statusCode = 200;
+  const { metadata } = input;
+  if (!Array.isArray(metadata)) {
+    inputs.metadata = [metadata];
+  }
```

Comment:
Do we need to handle the `else` part as well? Also, as we are sure that `metadata` will always be an object, we can skip the checking as well.

Reply:
yeah skipping it.